tests/fixtures/optimizer/tpc-ds/tpc-ds.sql

```diff
@@ -4529,7 +4529,7 @@ SELECT
   "item"."i_category" AS "i_category",
   "item"."i_class" AS "i_class",
   GROUPING("item"."i_category") + GROUPING("item"."i_class") AS "lochierarchy",
-  RANK() OVER (PARTITION BY GROUPING("item"."i_category") + GROUPING("item"."i_class"), CASE WHEN GROUPING("item"."i_class") = 0 THEN "item"."i_category" END ORDER BY SUM("store_sales"."ss_net_profit") / SUM("store_sales"."ss_ext_sales_price")) AS "rank_within_parent"
+  RANK() OVER (PARTITION BY GROUPING("item"."i_category") + GROUPING("item"."i_class"), CASE WHEN GROUPING("item"."i_class") = 0 THEN "item"."i_category" END ORDER BY SUM("store_sales"."ss_net_profit") / SUM("store_sales"."ss_ext_sales_price") ASC) AS "rank_within_parent"
```

Comment:
maybe the the optimizer canonicalize should get rid of asc

Reply:
Why should we get rid of it?

Reply:
because order by x == order by x asc

Reply:
should we do this in a separate PR?

Reply:
no, just do it here, it affects these tests

Reply:
done